utils.py

```diff
@@ -20,22 +22,34 @@ def get_zip(module_name, env):
     return output
 
 
-def get_dependencies(dir):
-    manifest_file = dir + '/__manifest__.py'
-    if not os.path.exists(manifest_file):
-        raise FileNotFoundError(f"Manifest file not found: {manifest_file}")
-    with open(manifest_file, encoding="utf-8") as f:
-        manifest = ast.literal_eval(f.read())
-    return set(manifest.get('depends', ['base']))
+def is_industry(module):
+    return os.path.exists(INDUSTRY_PATH + module)
+
+
+def get_dependencies(module_name):
+    in_list = {module_name}
+    other_dep = set()
+    industries = [module_name]
+    for industry in industries:
```

Comment:
Isn't it a bad practice to loop on a growing table? Shouldn't there be another list/set where we add new found dependencies and remove the current one?